docs/next.config.ts

```diff
@@ -273,5 +264,14 @@ export default withDocsInfra({
             { source: `/static/x/:rest*`, destination: 'http://0.0.0.0:3001/static/x/:rest*' },
           ];
         },
+        redirects: async () => {
+          return [
+            {
+              source: '/base-ui/',
+              destination: 'https://base-ui.com',
+              permanent: true,
+            },
+          ];
+        },
```

Comment:
Actually, this is wrong, we should never have a link, so we want it to crash in dev mode

```suggestion
```